src/CornerstoneViewport/CornerstoneViewport.js

```diff
@@ -803,23 +809,6 @@ class CornerstoneViewport extends Component {
     });
   };
 
-  startLoadingHandler = () => {
```

Comment:
I'm wondering if we should check if `cornerstoneTools.loadHandlerManager.getEndLoadHandler()` and `cornerstoneTools.loadHandlerManager.getStartLoadHandler()` exist when we are doing componentDidMount. If they don't exist, we could set them to do this setState behaviour. That way if the end user doesn't set them up and just uses one CornerstoneViewport component somewhere, the loading functionality would just work out of the box. Thoughts?

If they used two of these components though, they would need to move the handler logic to a higher level. Might be confusing to end users.

Reply:
Actually, it is already done at https://github.com/OHIF/react-viewerbase/blob/a94e3a9ff970185d220aa7381e48d4d82de004a9/src/LayoutChooser/LayoutManager.js#L63-L65